Cargo.toml

```diff
@@ -20,14 +21,16 @@ log = "0.4"
 env_logger = "0.5"
 lazy_static = "1.1.0"
 bytes = "0.4.9"
-ring = "0.12.1"
```

Comment:
Is untrusted still necessary? That was only for ring and according to it's author none of the inputs for the common api requires untrusted no more.